build.go

```diff
@@ -764,11 +766,47 @@ func rebuildAssets() {
 }
 
 func lazyRebuildAssets() {
-	if shouldRebuildAssets("lib/api/auto/gui.files.go", "gui") || shouldRebuildAssets("cmd/strelaypoolsrv/auto/gui.files.go", "cmd/strelaypoolsrv/gui") {
+	shouldRebuild := shouldRebuildAssets("lib/api/auto/gui.files.go", "gui") ||
+		shouldRebuildAssets("cmd/strelaypoolsrv/auto/gui.files.go", "cmd/strelaypoolsrv/gui")
+
+	if withNewGUI {
+		shouldRebuild = buildNewGUI() || shouldRebuild
+	}
```

Comment:
Could insert a notice here like `Not building experimental *insertFancyBetterName*; use -with-new-gui to build it`.